pdp/assignment.go

```diff
@@ -18,6 +19,14 @@ type AttributeAssignment struct {
 	e Expression
 }
 
+// AttribAssignFmt is the json marshal format of serialized AttributeAssignment
+type AttribAssignFmt struct {
+	Name      string
+	NameType  string
```

Comment:
Assignment target (lvalue) and expression to assign (rvalue) must have the same type so it's enough to have only Name, Type and... By the way at right side of the assignment we can have expression of any complexity. So it isn't correct to limit ourselves just with "Value". Here you should have marshalable "Expression" which creates something like this on JSON for `var x bool = s == "test"`:
```json
{
  "Name": "x",
  "Type": "boolean",
  "Value": {
    "equal": [
      {
        "attr": {
          "name": "s",
          "type": "string"
        }
      },
      {
        "value": {
          "type": "string",
          "content": "test"
        }
      }
    ]
  }
}
```